src/features/content-explorer/messages.js

```diff
@@ -143,6 +143,11 @@ const messages = defineMessages({
         description: "Results label for number of items on list when it's just 1",
         id: 'boxui.contentExplorer.result',
     },
+    folderNameTableHeader: {
```

Comment:
this could probably be column header instead of table header

```suggestion
    folderNameColumnHeader: {
```

Reply:
will remove completely so doesn't matter now